src/main/java/reposense/git/GitLog.java

```diff
@@ -14,6 +14,10 @@
  */
 public class GitLog {
 
+    /**
+     * Returns the git commit log info of {@code Author},
+     * in the repository specified in {@code config}
```

Comment:
This can be in a single line (max 120 chars in single line for `java` files). Also, no need to split into newlines if they are not separate points/sentences.

Reply:
The whole line is actually more than 120 chars.

Reply:
Whoops, then break the line at the word which passes the limit.